package-lock.json

```diff
@@ -199,6 +199,34 @@
         }
       }
     },
+    "@cypress/request": {
```

Comment:
lets remove this file from this PR as its unrelated

Reply:
ok